app/models/article.rb

```diff
@@ -236,23 +236,6 @@ def self.active_threads(tags = ["discuss"], time_ago = nil, number = 10)
     stories.pluck(:path, :title, :comments_count, :created_at)
   end
 
-  def self.active_eli5(time_ago)
```

Comment:
While I kept the custom logic on the #help tag, I thought that it wasn't needed in the same way with #ama.